### ISSUE TITLE
Rename executable and/or package to something that doesn't conflict with AWS CLI and isn't misleading

### ISSUE THREAD

**Issue by gffhcks** [eofs/aws#3]
I saw that there's an open issue to print a warning about this package not being the official CLI. I also saw that the issue hadn't been touched in 3 years, so this is probably just screaming into the void. But:

* Naming a published package "aws" when you're not actually publishing on behalf of AWS is highly misleading and arguably unethical (I don't think there was any harm intended here, but clearly plenty of users install this accidentally when they intend to install the official CLI)
* If someone is using this project, chances are they'll also want to use the official AWS CLI. This package installs an executable to your path with the same name, which can only possibly lead to problems (overwriting the executable, nondeterministic behavior based on paths, etc)

**Comment by arthurzenika** [comment]
This is a big problem from my point of view.

**Comment by arthurzenika** [comment]
See also https://github.com/eofs/aws/issues/2

**Comment by geyang** [comment]
@eofs any thoughts? I am happy to submit the PR to update the package name.

**Comment by dolguin-netquity** [comment]
Please please please rename this package!!

**Comment by bergpete** [comment]
@eofs any thoughts on this? I suspect there are at a minimum dozens if not 100s of developers each month who run into issues as a result of the chosen package name here

**Comment by sookcha** [comment]
Can we just talk to pypi moderator to remvoe or rename this package from pypi?

**Comment by geyang** [comment]
Just made a PEP 541 request folks. After seeing @sookcha's message in this thread. @arthurlogilab maybe you would help ping the `aws-cli` folks again? @aws-cli

https://github.com/aws/aws-cli/issues/4192

**Comment by sookcha** [comment]
> Just made a PEP 541 request folks. After seeing @sookcha's message in this thread. @arthurlogilab maybe you would help ping the `aws-cli` folks again? @aws-cli
> 
> [aws/aws-cli#4192](https://github.com/aws/aws-cli/issues/4192)

Agreed. Occupying and not maintaining the name “aws” from PyPi can lead to not only confusion but also many critical security issues.

**Comment by arthurzenika** [comment]
@geyang thanks for the PEP. I'm not sure what I can do for the `aws-cli` folks, can you be more specific ?

**Comment by jsta** [comment]
Perhaps the package could be reported as an issue to pypi support? https://github.com/pypa/pypi-support#issues-which-should-be-opened-here
